### ISSUE TITLE
[Feature request] thresholdAdjustment in other threshold.

### ISSUE THREAD

**Issue by zvezdochiot** [ScanTailor-Advanced/scantailor-advanced#59]
Hi @vigri , @lightsilverberryfox .

https://github.com/ScanTailor-Advanced/scantailor-advanced/blob/b4d8638bf96c564f434e35803dfd0fc8a432a2c6/src/core/filters/output/BlackWhiteOptions.cpp#L15
:question: Maybe add `thresholdAdjustment` in other threshold (Sauvola, Wolf, EdgePlus and BlurDiv) as an additional regulator?

https://github.com/ScanTailor-Advanced/scantailor-advanced/blob/b4d8638bf96c564f434e35803dfd0fc8a432a2c6/src/imageproc/Binarize.cpp#L87
to
```cpp
      if (int(grayLine[x]) < (threshold + m_thresholdAdjustment)) {
```

https://github.com/ScanTailor-Advanced/scantailor-advanced/blob/b4d8638bf96c564f434e35803dfd0fc8a432a2c6/src/imageproc/Binarize.cpp#L188
to
```cpp
      if ((grayLine[x] < lowerBound) || ((grayLine[x] <= upperBound) && (int(grayLine[x]) <  (threshold + m_thresholdAdjustment)))) {
```
and so on...

And add a slider to all widgets of thresholds?

PS: `lowerBound` and `upperBound` it is also desirable to use in all thresholds.

Complite `BinarizationOptionsWidget` for all threshold:
![BinarizationOptionsWidget ui](https://github.com/ScanTailor-Advanced/scantailor-advanced/assets/12370082/62051277-edaf-4c9a-beaa-390a78466f31)
[BinarizationOptionsWidget.ui.gz](https://github.com/ScanTailor-Advanced/scantailor-advanced/files/11750297/BinarizationOptionsWidget.ui.gz)

**Comment by vigri** [comment]
@zvezdochiot can I close this one? 
As to my understanding, this is "fixed" with #48

**Comment by zvezdochiot** [comment]
https://github.com/ScanTailor-Advanced/scantailor-advanced/commit/b0788fc5055ab8f7f7a6b5d727dd13eea1006212
